tensorrt_llm/_torch/pyexecutor/py_executor.py

```diff
@@ -210,8 +206,8 @@ def __init__(self,
         self.inflight_req_ids = ReqIdsSet()
 
         self.model_engine.warmup(self.resource_manager)
-        if self.draft_model_engine is not None:
-            self.draft_model_engine.warmup(self.resource_manager)
+        if self.drafter is not None:
```

Comment:
This needs a `hasattr(self.drafter, "draft_model_engine")` check (or, preferably, we add an `Optional[Drafter]` attribute to the `Drafter` base class. Then we can check `self.drafter is not None and self.drafter.draft_model_engine is not None`, which is a bit prettier).

Reply:
Thanks, I define the `self.draft_model_engine` via this:

```
self.draft_model_engine = getattr(self.drafter, "draft_model_engine", None)
```